### PR TITLE
MDAL update 0.6.0

### PR DESCRIPTION
**Pull request by vcloarec**
MDAL update 0.6.0

### CODE REVIEW

**Comment by nirvn** [comment]
Benefits?

**Comment by saberraz** [comment]
Would be good to have a short changelog here

**Comment by saberraz** [comment]
> Benefits?
you beat me to it!

**Comment by vcloarec** [comment]
change log since 0.5.0
- 1D mesh supported (UGRID and 3Di formats)
- support multi-mesh in the same file
- logger

**Comment by nyalldawson** [comment]
(also massive performance win 😜)

**Comment by nirvn** [comment]
@nyalldawson , massive performance win across the board or only for specific format(s)

@vcloarec , thanks, always useful to get a sense of new goodies you guys are bringing in :)

**Comment by PeterPetrik** [comment]
performance win is mainly for GRIB/netcdf (meteo) data

we already have 0.5.9x version (last beta release of 0.6.x) on master, this is just sync the master with the released MDAL.

File: external/mdal/cmake/FindSQLite3.cmake [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2007-2009 LuaDist.
```
Comment (PeterPetrik) [inline external/mdal/cmake/FindSQLite3.cmake:1]: this file should not be synced, it is already on cmake/ 

https://github.com/qgis/QGIS/blob/master/cmake/FindSqlite3.cmake

**Pull request merged**